src/Command/Resources/ResourcesSetCommand.php

```diff
@@ -395,7 +390,7 @@ protected function validateInstanceCount($value, $serviceName, $service, $limit)
         if ($limit !== null && $count > $limit) {
             return sprintf('The instance count <error>%d</error> exceeds the limit %d.', $count, $limit);
```

Comment:
Shouldn't this raise an `InvalidArgumentException`? Same at the 2 `return`s above.

Reply:
Indeed thanks!